agent/lib_aws_sdk_php.h

```diff
@@ -20,4 +30,71 @@ extern void nr_lib_aws_sdk_php_handle_version();
 extern void nr_lib_aws_sdk_php_add_supportability_service_metric(
     const char* service_name);
 
+#if ZEND_MODULE_API_NO >= ZEND_8_1_X_API_NO /* PHP8.1+ */
+/* Aside from service class and version detection, instrumentation is only
+ * supported with PHP 8.1+ */
+
+/*
+ * Purpose : Parses the QueueUrl to extract cloud_region, cloud_account_id, and
+ * destination_name.  The extraction sets all or none since the values from the
+ * same string and if it is malformed, it cannot be used.
```

Comment:
```suggestion
 * destination_name.  The extraction sets all or none since the values are from
 * the same string and if it is malformed, it cannot be used.
```

Reply:
in this PR: https://github.com/newrelic/newrelic-php-agent/pull/1003/commits/c1f163f7c43f1b731e1b94a515b0ec47cf4383f5